src/js/components/OptionalSettingsComponent.jsx

```diff
@@ -0,0 +1,63 @@
+var React = require("react/addons");
+
+var appValidator = require("../validators/appValidator");
+var FormGroupComponent = require("../components/FormGroupComponent");
+
+var OptionalSettingsComponent = React.createClass({
+  displayName: "OptionalSettingsComponent",
+
+  propTypes: {
+    errors: React.PropTypes.array,
+    model: React.PropTypes.object.isRequired
+  },
+
+  render: function () {
+    let model = this.props.model;
+    let errors = this.props.errors;
+    let helpMessage = "Comma-separated list of valid constraints. Valid constraint format is \"field:operator[:value]\".";
```

Comment:
Yeah I guess you're right. While technically there is no difference in a scenario like this one, I agree that we should _by convention_ only ever use `let` to signal an explicit block-scoping. :+1: